syntax = "proto3";

package fort.myrmidon.pb;

import "google/protobuf/timestamp.proto";
import "TagFamily.proto";
import "Measurement.proto";
import "AntMetadata.proto";

message AntShapeType {
	uint32 ID   = 1;
	string Name = 2;
};


message Experiment {
	string    name      = 2;
	string    author    = 3;
	string    comment   = 4;
	TagFamily tagfamily = 5;
	uint32    threshold = 6;
	double    tagsize   = 7;

	repeated MeasurementType customMeasurementTypes  = 21;

	repeated AntShapeType antshapetypes              = 31;

	repeated AntMetadataColumn antmetadata           = 41;
};
